test/unit/index.ts

```diff
@@ -447,150 +459,109 @@ describe('koa-body', () => {
       });
   });
 
-  describe('strict mode',  () => {
-    beforeEach( () => {
-      //push an additional, to test the multi query
-      database.users.push({ name: 'charlike' });
-    });
-
-    it('can enable strict mode',  (done) => {
-      app.use(koaBody({ strict: true }));
-      app.use(router.routes());
-
-      request(http.createServer(app.callback()))
-        .delete('/users/charlike')
-        .type('application/x-www-form-urlencoded')
-        .send({ multi: true })
-        .expect(204)
-        .end( (err, res) => {
-          if (err) return done(err);
-          assert(database.users.find(element => element.name === 'charlike') !== undefined);
-          done();
-        });
-    });
-
-    it('can disable strict mode',  (done) => {
-      app.use(koaBody({ strict: false }));
-      app.use(router.routes());
-
-      request(http.createServer(app.callback()))
-        .delete('/users/charlike')
-        .type('application/x-www-form-urlencoded')
-        .send({ multi: true })
-        .expect(204)
-        .end( (err, res) => {
-          if (err) return done(err);
-          assert(database.users.find(element => element.name === 'charlike') === undefined);
-          done();
-        });
-    });
-  });
-
-  describe('parsedMethods options',  () => {
-    beforeEach( () => {
+  describe('parsedMethods options', () => {
+    beforeEach(() => {
       //push an additional, to test the multi query
       database.users.push({ name: 'charlike' });
     });
 
-    it('methods declared are parsed',  (done) => {
-      app.use(koaBody({ parsedMethods: ['POST', 'PUT', 'PATCH', 'DELETE'] }));
+    it('methods declared are parsed', (done) => {
+      app.use(
+        koaBody({
+          parsedMethods: [
+            HttpMethodEnum.PATCH,
```

Comment:
We'll need to note the use of this in the CHANGELOG

Reply:
This is just the typescript approach, you can still pass in 'GET', 'POST', 'PUT' as normal strings

Reply:
but we can note them if you want. You decide 🥇

Reply:
done